packages/console/src/console-history/ConsoleHistoryResultErrorMessage.jsx

```diff
@@ -112,16 +108,18 @@ class ConsoleHistoryResultErrorMessage extends PureComponent {
             </button>
           </div>
         )}
-        <div
-          role="button"
-          tabIndex="0"
-          className="error-content"
-          onKeyPress={this.handleKeyPress}
-          onMouseDown={this.handleMouseDown}
-          onMouseMove={this.handleMouseMove}
-          onMouseUp={this.handleMouseUp}
-        >
-          {message}
+        <div className="error-content">
+          <button
```

Comment:
Keep this in a div so you can select text.

Check behaviour in Firefox to ensure right-click does not trigger.